lib/omiauth-rails.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
```

Comment:
I think there's a typo in the file name: it should be `omniauth-rails.rb` instead of `omiauth-rails.rb`